src/IceRpc.Protobuf/PipeReaderExtensions.cs

```diff
@@ -109,7 +109,6 @@ public static class PipeReaderExtensions
 
         if (readResult.Buffer.FirstSpan[0] == 1)
         {
-            reader.AdvanceTo(readResult.Buffer.GetPosition(5));
```

Comment:
This call to `AdvanceTo` was not required as the caller completes the reader upon exception.